### PR TITLE
more tests for POD

### PR DESCRIPTION
**Pull request by Flousen**

### CODE REVIEW

File: tests/test_pod.py [hunk 1]
```diff
@@ -5,23 +5,44 @@
...
+    def test_correlation_matirix(self):
         A = POD('correlation_matrix').reduce(snapshots)
         assert np.allclose(A, poddb, rtol=1e-03, atol=1e-08) or np.allclose(
             A, -1 * poddb, rtol=1e-03, atol=1e-08,)
-
-    def test_3(self):
-        A = POD('randomized_svd').reduce(snapshots)
+    
+    def test_correlation_matirix_savemem(self):
+        A = POD('correlation_matrix',save_memory = True).reduce(snapshots)
```
Comment (mtezzele) [inline tests/test_pod.py]: please add a space after the comma, and remove the spaces before/after the equal sign

File: tests/test_pod.py [hunk 2]
```diff
@@ -5,23 +5,44 @@
...
         assert np.allclose(A, poddb, rtol=1e-03, atol=1e-08) or np.allclose(
             A, -1 * poddb, rtol=1e-03, atol=1e-08,)
+
+    def test_randomized_svd(self):
+        A = POD('randomized_svd').reduce(snapshots)
+        #assert np.allclose(A, poddb, rtol=1e-03, atol=1e-08) or np.allclose(
```
Comment (mtezzele) [inline tests/test_pod.py]: Remove commented lines

File: ezyrb/pod.py [hunk 3]
```diff
@@ -116,24 +131,32 @@ def _rsvd(self, X):
 
     def _corrm(self, X):
         """
+        Truncated Singular Value Decomposition. calculated with correlation matrix.
+
+
```
Comment (mtezzele) [inline ezyrb/pod.py]: Remove unnecessary empty lines, one is ok

File: ezyrb/pod.py [hunk 4]
```diff
@@ -50,11 +50,19 @@ def singular_values(self):
         return self._singular_values
 
     def reduce(self, X):
+        """
+
+        :type: numpy.ndarray
+        """
         self._modes, self._singular_values = self.__method(X)
         print('AAAA', self._modes)
```
Comment (ndem0) [inline ezyrb/pod.py]: Remove the print

File: ezyrb/pod.py [hunk 5]
```diff
@@ -50,11 +50,19 @@ def singular_values(self):
         return self._singular_values
 
     def reduce(self, X):
+        """
+
```
Comment (ndem0) [inline ezyrb/pod.py]: Please add description of the method

File: ezyrb/pod.py [hunk 6]
```diff
@@ -50,11 +50,19 @@ def singular_values(self):
...
+        :type: numpy.ndarray
+        """
         self._modes, self._singular_values = self.__method(X)
         print('AAAA', self._modes)
         return self.modes.T.dot(X)
 
     def expand(self, X):
+        """
+
```
Comment (ndem0) [inline ezyrb/pod.py]: Please add description of the method

**Pull request merged**